src/Controls/src/Core/Platform/iOS/ModalWrapper.cs

```diff
@@ -152,7 +154,7 @@ public override void ViewDidLoad()
 		{
 			base.ViewDidLoad();
 			SetNeedsStatusBarAppearanceUpdate();
-			if (PlatformVersion.Supports(PlatformApis.RespondsToSetNeedsUpdateOfHomeIndicatorAutoHidden))
+			if (OperatingSystem.IsIOSVersionAtLeast(11) && PlatformVersion.Supports(PlatformApis.RespondsToSetNeedsUpdateOfHomeIndicatorAutoHidden))
```

Comment:
Does anyone know why this is such a weird property instead of just a OS version check? @PureWeen @StephaneDelcroix ?

Reply:
I think we can just delete all uses of `PlatformVersion`. That was an idea we had before `OperatingSystem` came into existence where we'd have something indicating what we are specifically checking for. But using `PlatformVersion` causes us to lose all the benefits of `OperatingSystem`. If there was a way to surface those benefits that'd be neat but that's probably not something we'll be able to do currently.

Reply:
OK, sounds like I need to remove `PlatformVersion.Supports(PlatformApis.RespondsToSetNeedsUpdateOfHomeIndicatorAutoHidden)` checks and lease `OperatingSystem` check